src/main/java/trello/ui/components/TopMenu.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package trello.ui.components;
+
+import core.selenium.util.WebDriverMethod;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import trello.ui.pages.BasePage;
+
+/**
+ * TopMenu class.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public class TopMenu extends BasePage {
+
+    @FindBy(name = "house")
+    private WebElement homeLink;
+
+    @FindBy(css = "div[class='TMI28E0KnYSK9p'] span[name='add']")
+    private WebElement addButton;
+
+    @FindBy(className = USER_CLASS_NAME)
+    private WebElement userButton;
+
+    private static final String USER_CLASS_NAME = "_24AWINHReYjNBf";
+
+    /**
+     * Constructor method for create an object of it's class.
+     */
+    public TopMenu() {
+
+    }
+
+    /**
+     * Gets the option home in the TopMenu.
+     *
+     * @return a object of Element.
+     */
+    public Element getHome() {
+        return new Link(homeLink);
+    }
+
+    /**
+     * Gets the option Add in the TopMenu.
+     *
+     * @return a object of Element.
+     */
+    public Element getAdd() {
+        Menu menuAdd = new Menu(addButton);
+        WebElement addBoardButton = driver.findElement(By.
+                cssSelector("button[class='_2jR0BZMM5cBReR'] span[name='board']"));
+        Link createBoard = new Link(addBoardButton);
+        WebElement addTeamButton = driver.findElement(By.
+                cssSelector("button[class='_2jR0BZMM5cBReR'] span[name='organization']"));
```

Comment:
You can also declare the locator as a By.

Reply:
Thank you this wrong was fixed.